index.js

```diff
@@ -199,6 +202,26 @@ module.exports = class Conf {
 		this.events.emit('change');
 	}
 
+	_ensureDirectory() {
+		// Ensure the directory exists as it could have been deleted in the meantime
+		// TODO: Use `fs.mkdirSync` `recursive` option when targeting Node.js 12
+		makeDir.sync(path.dirname(this.path));
+	}
+
+	_watch() {
+		this._ensureDirectory();
+
+		if (!fs.existsSync(this.path)) {
+			writeFileAtomic.sync(this.path, JSON.stringify({}, null, '\t'));
+		}
+
+		fs.watch(this.path, debounceFn(eventType => {
+			if (eventType === 'change') {
+				this.events.emit('change');
+			}
+		}, {wait: 100}));
```

Comment:
You also need the `persistence: false` option. Please read the `fs.watch` docs more closely.